test/__helpers/workspace.ts

```diff
@@ -88,7 +93,7 @@ async function doCreateWorkspace(config: Options): Promise<Workspace> {
           nexus: dir.pathRelativeTonexusFuture,
         },
         scripts: {
-          postinstall: 'yarn -s link nexus && chmod +x node_modules/.bin/nexus',
+          postinstall: 'yarn -s link nexus',
```

Comment:
I assume that this is required to run `> nexus` on Unix. Thoughts on creating a postinstall script like in `nexus-plugin-prisma`